vars/virtualenv.groovy

```diff
@@ -0,0 +1,14 @@
+def create(String python, String path = "venv", boolean installRequirements = true) {
```

Comment:
This function does too much to my taste. I think it should only do one thing: create a virtualenv. I am fine with adding a function to simplify calling `pip install -r requirements.txt`. but it doesn't belong in this one.